k2/csrc/fsa_equivalent_test.cc

```diff
@@ -149,6 +149,19 @@ TEST(FsaEquivalent, IsWfsaRandEquivalent) {
                                                   c_weights.data());
     EXPECT_FALSE(status);
   }
+
+  // check equivalence with beam
+  {
+    bool status = IsRandEquivalent<kMaxWeight>(a, a_weights.data(), b,
+                                               b_weights.data(), 4);
```

Comment:
4.0 ?  If this is the beam.  And why this value?

Reply:
ok, will chaned to 4.0. It just a value I get for the testing FSA (will prune one path in the input fsa with this value